python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -320,6 +320,9 @@ def _scheduling_loop_step(self, topology: Topology) -> bool:
         # Update the progress bar to reflect scheduling decisions.
         for op_state in topology.values():
             op_state.refresh_progress_bar(self._resource_manager)
+        # Refresh the global progress bar to update elapsed time progress.
+        if self._global_info and self._global_info._bar:
+            self._global_info._bar.refresh()
```

Comment:
Should we add a `refresh` method to `ProgressBar`? My impression is that this is breaking an abstraction barrier by accessing the private `ProgressBar._bar` attribute?

Reply:
yeah good point, that will help clean up the logic as well. added it